scripts/render_tags.py

```diff
@@ -54,6 +54,21 @@ def format_path(metadata):
         return "Unknown"
 
 
+def is_base_class(symbol):
```

Comment:
This is so the base classes don't render with massive preset tables in the `from_preset()` function. I am somewhat torn here. For `CausalLM` and `Classifier` it is a little useful to see the list of builtins they support. For `Backbone` and `Tokenizer` it is less useful since we just list every preset in the whole library.